chrome/content/prefs.xul

```diff
@@ -11,15 +11,25 @@
   <prefpane id="otr-pane">
 
     <preferences id="otr-preferences">
+      <preference id="extensions.otr.preferences.selectedTabIndex"
+                  name="extensions.otr.preferences.selectedTabIndex"
+                  type="int"/>
```

Comment:
Not really an issue, but what happens when CONIKS is the selected tab and then it gets disabled.

This selected tab is just so that it remembers which tab you were on last after closing the window?  Do we really need that?